UI/src/view/point_asset/massTransitMarker.js

```diff
@@ -312,16 +379,24 @@
         }
       }
 
-      if(data.stopTypes[0] == 6)
+      if(data.stopTypes[0] == 6 || data.stopTypes[0] == 7 )
         direction = '';
 
       var styles = [];
       styles = styles.concat(createDirectionArrowStyle());
       styles = styles.concat(createStickStyle());
-      styles = styles.concat(createSelectionBackgroundStyle(data.stopTypes, name+direction));
-      styles = styles.concat(createStopBackgroundStyle(data.stopTypes, IMAGE_MARGIN, validityPeriod));
-      styles = styles.concat(createStopTypeStyles(data.stopTypes, IMAGE_MARGIN));
-      styles = styles.concat(createTextStyles(data.stopTypes, nationalId, name, direction, IMAGE_MARGIN));
+
+      /* Due the impact of the order of the concats*/
+      if (data.stopTypes.length > 0 && data.stopTypes[0] == 7) {
```

Comment:
This "data.stopTypes.length > 0" it's used to verify if exist stopTypes?
And if it is used for that whats happens in case of a data object doesn't have stopTypes?

Reply:
Ok, I will only modify to use !_.isEmpty(data.stopTypes) instead data.stopTypes.length > 0

Reply:
Done